edit_course/operations/configure.py

```diff
@@ -429,3 +464,29 @@ def configure_content(instance, url):
             category.delete()
 
     return errors
+
+def get_target_category(course, category):
+
+    if not category:
+        return None, _("ExerciseCollection object requires collection_category.")
+
+
+    course_name, instance_name = course.split(";")
```

Comment:
Doesn't support colon in course name or instance name.

Url part is probably better as there is strict set of allowed letters.